packages/itwin/measure-tools/package.json

```diff
@@ -105,12 +105,13 @@
     "source-map-loader": "^1.0.0",
     "typescript": "^4.4.0",
     "url-loader": "2.1.0",
-    "webpack": "4.44.2",
-    "webpack-cli": "^3.3.8"
+    "webpack": "5.76.0",
+    "webpack-cli": "^5.0.1"
   },
   "dependencies": {
     "@bentley/icons-generic-webfont": "^1.0.29",
-    "@itwin/core-telemetry": "^3.7.0"
+    "@itwin/core-telemetry": "^3.7.0",
+    "@itwin/core-i18n": "3.7.1"
```

Comment:
if youre only using this for tests make this a dev dep

Reply:
this should be ^3.7.0

Reply:
changed